Standardise component structure (#150)
sa-tre/satre-specification: docs/source/contributing/index.md

Issue templates have been designed for common issue types to help collect the most important information and present it in a clear, consistent way.
It is possible, however, to open a blank issue if none of the templates are suitable.

> While we encourage opening issues, we understand that some may be more comfortable contributing ideas in other ways, such as through discussions and notes at SATRE Collaboration Cafés.

Reviewer: ```suggestion
While we encourage opening issues, we understand that some may be more comfortable contributing ideas in other ways. 
We support other methods of contribution such as discussions and notes taken at SATRE Collaboration Cafés.
```

The SATRE Team will aim to collate ideas and draft issues that welcome further discussion and attribute those involved in initial discussions.
The SATRE Team will try to capture the ideas as accurately as possible, in good faith, and be guided by the SATRE Community to correct any misconceptions.

When ready, changes will be proposed in pull requests.
Similarly to issues, there is a pull request template.

> This template prompts contributors to include important details, which helps explain the contribution and makes triage and review easier.

Author: I think the correct time to use a comma before "which" is when it introduces a clause that is not essential to parsing the sentence. In this case I think I'd prefer to rephrase.

> This template prompts contributors to include important details, which helps explain the contribution and makes triage and review easier.

Author: ```suggestion
This template prompts contributors to include important details in order to explain the contribution and make triage and review easier.
```

> This template prompts contributors to include important details, which helps explain the contribution and makes triage and review easier.

Reviewer 2: I think it's correct here, fwiw
